google/cloud/speech/cloud_speech.yaml

```diff
@@ -11,17 +11,18 @@ documentation:
     Google Cloud Speech API.
 
 apis:
-- name: google.cloud.speech.v1.Speech
+- name: google.cloud.speech.v1beta1.Speech
+- name: google.longrunning.Operations
```

Comment:
Agreed. How do we want to do this? @pongad did a `toolkit` change to ignore streaming methods; shall we do the same for LRO methods? Alternatively, we can comment it out in the proto and configs, but that's messier. (I prefer commenting out vs. deleting in this case, because it makes it clear that we didn't forget to include it, and we can associate a TODO comment explaining under what conditions we'll un-comment it)